setup

```diff
@@ -17,7 +17,7 @@ if test "$(uname)" = "Darwin"; then
     brew upgrade ansible
   fi
 elif test "$(uname)" = "Linux"; then
-  if test -f /etc/lsb-release -a ! "$(which ansible)"; then
+  if test -f /etc/lsb-release && -a ! "$(which ansible)"; then
```

Comment:
hmm, some shells may indeed not recognize the `-a`, but AFAIK bash wasn't one of them.

Anyways, removing the `-a` will turn it into a valid `if` statement again, yes :)